### PR TITLE
Minor validation/logging and prep for 3.0.4 release

### PR DESCRIPTION
**Pull request by jakelandis**
Just some random improvements I found while testing out #11.

### CODE REVIEW

File: lib/logstash/inputs/meetup.rb [hunk 1]
```diff
@@ -44,10 +44,11 @@ def register
 	addon = "venue_id=#{ @venueid }"
     else
     # None Selected, raise an error
+    raise "Configuration error! -  Must have one of `urlname`, `venue_id`, or `group_id` defined"
```
Comment (colinsurprenant) [inline lib/logstash/inputs/meetup.rb]: indentation?

Comment (jakelandis) [reply lib/logstash/inputs/meetup.rb]: @colinsurprenant - good eye..fixed.  The whole file had funky formatting, so I just ran my formatter on it. https://github.com/logstash-plugins/logstash-input-meetup/pull/12/commits/c8324c39247780808e833f3ca9018a3cf4c1d7a9 => format changes only.

**Comment by colinsurprenant** [comment]
LGTM

**Comment by jakelandis** [comment]
@colinsurprenant - thanks!

**Pull request closed without merging**